src/components/tabset/tabset.component.spec.ts

```diff
@@ -116,12 +117,35 @@ describe('Tabset Component', () => {
         expect(component.onTabDeselect).toHaveBeenCalled();
     });
 
+    it('should change to new tab when tab is programmatically selected', () => {
+
```

Comment:
It would be good to have 2 separate tests one that tests using the index as you have done and then another with the TabComponent, so we can ensure both work.

Reply:
Good thinking.

Reply:
Actually, how would I go about doing that? Just two separate buttons using the two different methods?

Reply:
Since this is a unit test you can just call the function directly, no need for buttons.